### DISCUSSION TITLE
error on Installation about waitress==2.1.1

### DISCUSSION THREAD

**Discussion by krypto-san** [FabCx/TradingView-Webhook-Bot#89] [Q&A] [answered]
Hey 

I am encountering an error at requirement.txt installation. I get the following error. 

"No matching distribution found for waitress==2.1.1 "

![Screenshot_2](https://user-images.githubusercontent.com/84674045/163737228-13e2edb3-4b2f-4c26-8ba4-ec811da3b31b.png)

How do I solve this error? 

Regards .

**Comment by fabston** [comment, marked as answer]
Hello,

Inside your environment please run the following command `pip install waitress`. 

If that is also not working try what is mentioned [here](https://github.com/fabston/TradingView-Webhook-Bot/issues/41#issuecomment-810530573).

**Reply by krypto-san** [reply]
Thanks for the quick reply. i ran the command and it install waitress 2.0.0 . is it okay?

**Reply by fabston** [reply]
Yes that’s totally fine.
